charts/hono/templates/hono-service-command-router/hono-service-command-router-secret.yaml

```diff
@@ -92,9 +92,7 @@ stringData:
         {{- required ".Values.adapters.commandAndControlSpec MUST be set if example AMQP Messaging Network is disabled" .Values.adapters.commandAndControlSpec | toYaml | nindent 8 }}
       {{- end }}
       {{- end }}
-      {{- if has "kafka" .Values.messagingNetworkTypes }}
-        {{- include "hono.kafkaMessagingConfig" ( dict "dot" . "component" $args.component "kafkaMessagingSpec" .Values.commandRouterService.hono.kafka ) | indent 6 }}
-      {{- end -}}
+      {{- include "hono.messagingNetworkClientConfig" ( dict "dot" . "component" $args.component "kafkaMessagingSpec" .Values.commandRouterService.hono.kafka ) | nindent 6 }}
       {{- if and (not (has "amqp" .Values.messagingNetworkTypes)) (not (has "kafka" .Values.messagingNetworkTypes)) -}}
         {{- required "Property messagingNetworkTypes MUST contain 'amqp' and/or 'kafka'" nil }}
       {{- end }}
```

Comment:
This block can now be removed as well - it is already contained in the invoked `hono.messagingNetworkClientConfig` named template.